### ISSUE TITLE
[Bug Report] About 1.1.0-beta.18 Icon cannot be displayed in image

### ISSUE THREAD

**Issue by duzhipeng** [element-plus/element-plus#3729]
### Element Plus version
1.1.0-beta.18

### OS/Browsers version
macOS 11.6/Safari 15.0

### Vue version
3.2.19

### Reproduction Link
https://jsfiddle.net

### Steps to reproduce
核心的问题主要是根据新文档使用SVG ICON和全新的SCSS定制主题方式，诸如 Image 组件中的 Image Preview 是官方封装好的，然后里面有图标用的又是原来字体的方式我真不好改。如果不用SVG ICON按老文档的方式SCSS定制主题，定制则不能生效。这样陷入了两难的境地。
The core problem is to use SVG ICON and brand new SCSS to customize themes according to the new document. For example, the Image Preview in the Image component is officially packaged, and the icons inside are the original fonts. I really can’t change it. If you do not use SVG ICON to customize the theme according to the old document SCSS, the customization will not take effect. This is in a dilemma.
<img width="1413" alt="QQ20210929-152234@2x" src="https://user-images.githubusercontent.com/3648161/135224440-52abe678-08e4-4fe2-bce1-7918e4eef20c.png">

### What is Expected?
 Image 组件中的 Image Preview 图标基于最新1.1.0-beta.18的文档，使用新的SCSS定制主题方式及SVG ICON的前提下，能够正常显示放大、旋转等功能按钮的图标。
The Image Preview icon in the Image component is based on the latest 1.1.0-beta.18 document. With the new SCSS customized theme and SVG ICON, the icons for function buttons such as magnification and rotation can be displayed normally.

### What is actually happening?
基于1.1.0-beta.18 Image 组件中的 Image Preview 中的功能图标都不能显示。
Based on 1.1.0-beta.18, none of the function icons in the Image Preview in the Image component can be displayed.

**Comment by JeremyWuuuuu** [comment]
@duzhipeng 我们正在做 SVG 的全量迁移，你可以看一下这个 PR #3492 当这个 PR merge 到 dev 之后，我们会有一个新版本，里面到时候 Element Plus 提供的 index.css 将不在默认包含全量的 font icon 图标。进度挺快的，应该不会再花很久时间，我初步估计 ~10月15左右。

**Comment by duzhipeng** [comment]
> @duzhipeng 我们正在做 SVG 的全量迁移，你可以看一下这个 PR #3492 当这个 PR merge 到 dev 之后，我们会有一个新版本，里面到时候 Element Plus 提供的 index.css 将不在默认包含全量的 font icon 图标。进度挺快的，应该不会再花很久时间，我初步估计 ~10月15左右。

谢谢大佬！
Thx BOSS!

**Comment by JeremyWuuuuu** [comment]
@duzhipeng 现在 Element Plus 正在做一系列的改动，个人感觉处于一个阵痛期，非常感谢大家的理解和支持，我们一定会把这个项目做好的。:bow:

**Comment by adiramardiani** [comment]
"We will definitely make this project a success."
Yes thank you @JeremyWuuuuu , element plus is very promising, thank you to all developers, for your hard work 👍 ❤️ 

Btw currently font icon is not found, I rollback to v ^1.1.0-beta.17 all will be fine
fonts/element-icons.woff net::ERR_ABORTED 404 (Not Found)

can you share temporary solution with latest version ?

**Comment by duzhipeng** [comment]
> "We will definitely make this project a success." Yes thank you @JeremyWuuuuu , element plus is very promising, thank you to all developers, for your hard work 👍 ❤️
> 
> Btw currently font icon is not found, I rollback to v ^1.1.0-beta.17 all will be fine fonts/element-icons.woff net::ERR_ABORTED 404 (Not Found)
> 
> can you share temporary solution with latest version ?

Hello adiramandiani
In 1.1.0-beta.17, the font is normal. When I submitted the question before, I tried to roll back to this version and everything was normal.
The problem may be: if you use 1.1.0-beta.17, you must follow the old version of the document: https://doc-archive.element-plus.org/#/en-US/component/custom-theme, do it SCSS customization, the key is $--font-path:'~element-plus/lib/theme-chalk/fonts'
If you don't use SCSS to customize, then unfortunately I don't know how to help you. I can only make sure to customize the theme in SCSS and follow the old documents, it must be fine.
Oh, and if you use Vite, you must also use vite-plugin-element-plus in the old version of the document instead of unplugin-element-plus in the new version of the document, otherwise it will be abnormal.
